great-idea/services.html

```diff
@@ -157,7 +157,7 @@ <h2>Android Development</h2>
 
     </section>
 
-    <hr class="divider" />
+    <hr class="divider services-page-divider" />
```

Comment:
I see you are using hr tags in your html. Given that these horizontal lines are purely a design choice, I would suggest keeping them out of your html. You can accomplish the same thing by using either a border-top or border-bottom property in your css.